infra.bs

```diff
@@ -296,30 +296,30 @@ for working with them, in order to create common ground.
 <h3 id=lists>Lists</h3>
 
 <p>A <dfn export>list</dfn> is a specification type consisting of a finite ordered sequence of
-items.
+<dfn for=list lt=item export>items</dfn>.
```

Comment:
Yeah, it wasn't clear to me whether you wanted to create some sort of inheritance structure or "expose" the term on each one of the list types.